docker/docker-compose.yml

```diff
@@ -467,10 +467,10 @@ services:
       - POSTGRES_PORT=${POSTGRES_PORT}
       - POSTGRES_DB=${POSTGRES_DB}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-      - DATABASE_URL=ecto://postgres:${POSTGRES_PASSWORD}@db:${POSTGRES_PORT}/_supabase
+      - DATABASE_URL=ecto://postgres:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/_supabase
       - CLUSTER_POSTGRES=true
-      - SECRET_KEY_BASE=UpNVntn3cDxHJpq99YMc1T1AQgQpc8kfYTuRgBiYa15BLrx8etQoXz3gZv1/u2oq
-      - VAULT_ENC_KEY=your-encryption-key-32-chars-min
+      - SECRET_KEY_BASE=${POOLER_SECRET_KEY_BASE}
+      - VAULT_ENC_KEY=${POOLER_VAULT_ENC_KEY}
```

Comment:
**issue**

Those two values aren't available in the `.env.example` we'll need to add them in it with their default value so the `cp .env.example .env && docker-compose up` can start the stack with default settings.